### PR TITLE
Move some URLs to HTTPS

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: proposed/security-reporting-process-meta.md [hunk 1]
```diff
@@ -45,22 +45,22 @@ for how security vulnerabilities go from discovery to fixing to public
...
+* https://framework.zend.com/security
+* https://www.yiiframework.com/security
 * https://www.drupal.org/security
-* http://codex.wordpress.org/FAQ_Security
-* http://www.sugarcrm.com/page/sugarcrm-security-policy/en
-* http://typo3.org/teams/security/
+* https://codex.wordpress.org/FAQ_Security
+* https://www.sugarcrm.com/legal/security-policy
+* https://typo3.org/teams/security/
 * http://cakephp.org/development
```
Comment (fabpot) [inline proposed/security-reporting-process-meta.md]: I have not changed this one as it does not seem correct to me; the page is not related to security.

File: proposed/security-reporting-process-meta.md [hunk 2]
```diff
@@ -45,22 +45,22 @@ for how security vulnerabilities go from discovery to fixing to public
...
-* http://developer.joomla.org/security.html
-* http://wiki.horde.org/SecurityManagement
-* http://www.revive-adserver.com/support/bugs/
-* http://magento.com/security
-* http://www.apache.org/security/committers.html
+* https://www.concrete5.org/developers/security/
+* https://developer.joomla.org/security.html
+* https://wiki.horde.org/SecurityManagement
+* https://www.revive-adserver.com/support/bugs/
+* https://magento.com/security
```
Comment (fabpot) [inline proposed/security-reporting-process-meta.md]: This page is more about getting security patches and less about the process.

File: proposed/security-reporting-process-meta.md [hunk 3]
```diff
@@ -45,22 +45,22 @@ for how security vulnerabilities go from discovery to fixing to public
...
 
-* http://symfony.com/security
-* http://framework.zend.com/security/
-* http://www.yiiframework.com/security/
+* https://symfony.com/security
+* https://framework.zend.com/security
+* https://www.yiiframework.com/security
 * https://www.drupal.org/security
-* http://codex.wordpress.org/FAQ_Security
-* http://www.sugarcrm.com/page/sugarcrm-security-policy/en
```
Comment (fabpot) [inline proposed/security-reporting-process-meta.md]: This is a 404 now, so I've updated to the new one.

File: proposed/security-reporting-process-meta.md (hunk 1 above)
Comment (pierredup) [reply proposed/security-reporting-process-meta.md]: The security page is available at https://book.cakephp.org/3.0/en/contributing/tickets.html#reporting-security-issues

Comment (fabpot) [reply proposed/security-reporting-process-meta.md]: updated, thanks

**Pull request merged**